text/0000-clarity-api-description.md

```diff
@@ -47,18 +41,8 @@ This type of transition, though, should have a URL and a method.
 
 We should add a section that defines what media types the server will respond with.
 
-## Add Schema to Resource
-
-A resource should be able to have its own schema, whereas now it cannot.
-
```

Comment:
So are we never going to address this, or just not going to address it as part of this RFC? 

I believe Swagger [allows for this](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#path-item-object).